tests/py/test_api_multi_gpu.py

```diff
@@ -0,0 +1,97 @@
+import unittest
+import trtorch
+import torch
+import torchvision.models as models
+
+from model_test_case import ModelTestCase
+
+gpu_id = 1
+class TestCompile(ModelTestCase):
+
+    def setUp(self):
+        self.gpu_id = gpu_id
```

Comment:
Check during setup if there is more than one GPU (you can probably do this through pytorch apis). If there is not then error out with a message saying this test is not valid on this platform.

Reply:
Dropped

Reply:
Added the device_count test